x-pack/test/observability_functional/apps/observability/alerts/index.ts

```diff
@@ -232,10 +237,20 @@ export default ({ getService }: FtrProviderContext) => {
         await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
       });
 
-      it('Is disabled when a user has only read privilages', async () => {
+      // Note: By adding "View Rule Details" button in the menu, we made "more actions" button always visible and enabled.
+      // Therefore the below test case and the code covered by it became redundant.
+      // In case of we may need it later, we didn't remove the code but disabled the test case since it's always invalid.
```

Comment:
I'd suggest to remove the code if not needed. We can always resume it later from history if it becomes useful again.

Reply:
Actually after adding a null check for  linkToRule, that code does not look that wrong. But test case is still useless...
Removing it.